gulpfile.js

```diff
@@ -677,24 +719,22 @@ function release_rpm(arch, appDirectory, done) {
     // The buildRpm does not generate the folder correctly, manually
     createDirIfNotExists(RELEASE_DIR);
 
-    const regex = /-/g;
-
     const options = {
-            name: pkg.name,
-            version: pkg.version.replace(regex, '_'), // RPM does not like release candidate versions
+            name: metadata.name,
+            version: metadata.version.replace(NAME_REGEX, '_'), // RPM does not like release candidate versions
```

Comment:
Is this comment still needed?

Reply:
I haven't heard anything about dashes ('-') now being supported in RPM version names. Have you?

Reply:
I see, I misunderstood this comment that I think this RPM is "Revolutions Per minute". It's the Red-Hat Package Manager, right?